src/main/java/org/folio/helper/BindHelper.java

```diff
@@ -73,6 +79,20 @@ private Map<String, List<Piece>> updatePieceRecords(Map<String, List<Piece>> pie
     return piecesGroupedByPoLine;
   }
 
+  private Map<String, List<Piece>> updateItemStatus(Map<String, List<Piece>> piecesGroupedByPoLine,
+                                                            RequestContext requestContext) {
+    List<String> itemIds = piecesGroupedByPoLine.values().stream().flatMap(List::stream)
+        .map(Piece::getItemId).toList();
+    inventoryItemManager.getItemRecordsByIds(itemIds, requestContext)
+      .compose(items -> {
+        items.forEach(item ->
+          item.put(ITEM_STATUS, new JsonObject().put("date", new Date()).put("name", "Unavailable"))
```

Comment:
we have an enum with item statuses that should be used

Reply:
Thanks, used